modular_skyrat/modules/modular_modsuit/module_of_holding.dm

```diff
@@ -0,0 +1,51 @@
+/obj/item/mod/module/storage/bluespace/boh/inert //Inert state of the BOH mod. Requires a refined bluespace core.
```

Comment:
this pathing is bad

Reply:
yeaaah really need to type this to /obj/item/mod/module/storage/boh/inert or something.